src/index.html

```diff
@@ -0,0 +1,240 @@
+                </div>
+             </div>
+          </div>
+          <div class="card">
+             <div class="card-header">
+                <a class="collapsed card-link" data-toggle="collapse" href="#collapseThree">
+                HTML-Prototyping mit Baumeister Teil 3 &#x2013; Handlebars Templates
+                </a>
+             </div>
+             <div id="collapseThree" class="collapse" data-parent="#accordion">
+                <div class="card-body">
+                   <div style="padding:56.25% 0 0 0;position:relative"><iframe src="https://player.vimeo.com/video/268576053?color=75B7DD&amp;title=0&amp;byline=0&amp;portrait=0" style="position:absolute;top:0;left:0;width:100%;height:100%" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen></iframe></div>
+                   <script src="https://player.vimeo.com/api/player.js"></script>
+                </div>
+             </div>
+          </div>
+       </div>
+    </div>
+ </div>
+ <div class="row">
+    <div id="anchor-install" class="install gray col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>Quick install guide</h2>
+       <h6>For those already using Node, Gulp and stuff.</h6>
+       <div class="viaYeoman">
+          <p>via <strong>Yeoman</strong></p>
+          <pre class="language-bash"><code class="language-bash">$ npm <span class="token function">install</span> -g yo
+$ npm <span class="token function">install</span> -g generator-baumeister
+$ yo baumeister
+$ npm run tasks
+       </code></pre>
+          <p>See https://github.com/<wbr>micromata/generator-baumeister for details.</p>
+       </div>
+       <div class="viaGit">
+          <p>via <strong>Git</strong></p>
+          <pre class="language-bash"><code class="language-bash">$ <span class="token function">git</span> clone https://github.com/micromata/baumeister.git
+$ <span class="token function">cd</span> baumeister
+$ npm <span class="token function">install</span>
+$ npm run tasks
+       </code></pre>
+          <p>See https://github.com/<wbr>micromata/baumeister for details.</p>
+       </div>
+    </div>
+ </div>
+ <div class="row">
+    <div id="anchor-socials" class="keepintouch white col-lg-12 col-md-12 col-ms-12 col-xs-12">
+       <h2>Our Socials</h2>
+       <h6>Baumeister on...</h6>
+       <div class="our_socials">
+          <div class='wrp'>
+             <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/baumeister_io" class='icon-twitter icon'>
+                <i class='fab fa-twitter'></i>
+                <p>Twitter</p>
+             </a>
+             <a target="_blank" rel="noopener noreferrer" href="https://github.com/micromata/baumeister" class='icon icon-github'>
+                <i class='fab fa-github'></i>
+                <p>Github</p>
+             </a>
+             <a target="_blank" rel="noopener noreferrer" href="https://labs.micromata.de/projects/baumeister.html" class='icon icon-labs'>
```

Comment:
Url hat sich nach Labs Relaunch geändert.
```suggestion
             <a target="_blank" rel="noopener noreferrer" href="https://labs.micromata.de/baumeister/" class='icon icon-labs'>
```